tests/test_utils_flock.py

```diff
@@ -54,6 +54,7 @@ def _wait_and_set(event: Event, sleep_seconds: float):
     event.set()
 
 
+@pytest.mark.flaky(retries=3)
```

Comment:
You don't need these since it's specified in the overall config. This is only if you want to override the config for it or want to enable it when it's not enabled in the top-level config